scripts/create_istio_system.sh

```diff
@@ -33,3 +33,14 @@ else
 fi
 istioctl manifest generate -i $ISTIO_NAMESPACE ${ISTIO_FILES[@]/#/-f } > /app/generated-manifest.yaml
 istioctl verify-install -i $ISTIO_NAMESPACE -f /app/generated-manifest.yaml
+
+if kubectl get namespace cattle-dashboards; then
+  kubectl delete configmap -n cattle-dashboards -l istio_dashboard=1
+  for dashboard in /usr/local/app/dashboards/*.json; do
+    name="istio-dashboard-$(basename ${dashboard} ".json")"
+    sed -i 's/${DS_PROMETHEUS}/Prometheus/g' ${dashboard}
```

Comment:
How does the `${DS_PROMETHEUS}` part work ?

Reply:
It replaces the datasource input placeholder in the Grafana dashboard with the actual datasource name in rancher-monitoring. I added a comment to explain this.